Content.Client/GameObjects/Components/Chemistry/ReagentDispenserBoundUserInterface.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Robust.Client.GameObjects.Components.UserInterface;
+using Robust.Client.UserInterface.Controls;
+using Robust.Shared.GameObjects.Components.UserInterface;
+using static Content.Shared.GameObjects.Components.Chemistry.SharedReagentDispenserComponent;
+
+namespace Content.Client.GameObjects.Components.Chemistry
+{
+    /// <summary>
+    /// Initializes a <see cref="ReagentDispenserWindow"/> and updates it when new server messages are received.
+    /// </summary>
+    public class ReagentDispenserBoundUserInterface : BoundUserInterface
+    {
+        private ReagentDispenserWindow _window;
+        private ReagentDispenserBoundUserInterfaceState _lastState;
+
+        public ReagentDispenserBoundUserInterface(ClientUserInterfaceComponent owner, object uiKey) : base(owner, uiKey)
+        {
+
+        }
+
+        /// <summary>
+        /// Called each time a dispenser UI instance is opened. Generates the dispenser window and fills it with
+        /// relevant info. Sets the actions for static buttons.
+        /// <para>Buttons which can change like reagent dispense buttons have their actions set in <see cref="UpdateReagentsList"/>.</para>
+        /// </summary>
+        protected override void Open()
+        {
+            base.Open();
+
+            //Setup window layout/elements
+            _window = new ReagentDispenserWindow()
+            {
+                Title = "Reagent dispenser",
```

Comment:
You should localize all the strings in here with `ILocalizationManager`.

Reply:
Done. Did the same for strings in `ReagentDispenserWindow` and any other strings from this PR I could find.